<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI_hasil_kuis</class>
 <widget class="QMainWindow" name="UI_hasil_kuis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #90caf9, stop:1 #a6d4fa);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="kalimat_penyemangat">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Lato Heavy</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Semangat</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="kalimat_skor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Skor Kuismu = 7</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>30</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Lato</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Soal dengan Jawaban Salah</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>770</width>
             <height>391</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="soal_kuis">
              <property name="font">
               <font>
                <family>Lato</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Protokol yang umum digunakan untuk mengetahui physical address dari sebuah komputer dengan IP address tertentu disebut sebagai</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="jawaban_pengguna">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Kamu menjawab = </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="jawaban_benar">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Jawaban yang benar = </string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <property name="topMargin">
               <number>20</number>
              </property>
              <item row="0" column="0">
               <widget class="QPushButton" name="sebelumBtn">
                <property name="styleSheet">
                 <string notr="true">padding: 4px 15px;
color: rgb(37, 179, 255);
font-size: 11px;
text-align: center;
cursor: pointer;
outline: none;
color: #fff;
background-color: #306ea8;
border: none;
border-radius: 15px;
box-shadow: 0 9px #999;</string>
                </property>
                <property name="text">
                 <string>Soal Sebelumnya</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="sesudahBtn">
                <property name="styleSheet">
                 <string notr="true">padding: 4px 15px;
color: rgb(37, 179, 255);
font-size: 11px;
text-align: center;
cursor: pointer;
outline: none;
color: #fff;
background-color: #306ea8;
border: none;
border-radius: 15px;
box-shadow: 0 9px #999;</string>
                </property>
                <property name="text">
                 <string>Soal Sesudahnya</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>30</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="kembali_hasilKuisBtn">
          <property name="styleSheet">
           <string notr="true">padding: 4px 15px;
font-size: 11px;
text-align: center;
cursor: pointer;
outline: none;
color: #fff;
background-color: #3949AB;
border: none;
border-radius: 15px;
box-shadow: 0 9px #999;</string>
          </property>
          <property name="text">
           <string>Kembali</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>resource/1x/Asset 2.png</normaloff>resource/1x/Asset 2.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
